src/frontend/contexts/AppProviders.tsx

```diff
@@ -39,26 +42,32 @@ export const AppProviders = ({
   return (
     <IntlProvider>
       <QueryClientProvider client={queryClient}>
-        <GestureHandlerRootView style={{flex: 1}}>
-          <TrackTimerContextProvider>
-            <GPSModalContextProvider>
-              <ServerLoading messagePort={messagePort}>
-                <LocalDiscoveryProvider value={localDiscoveryController}>
-                  <ApiProvider api={mapeoApi}>
-                    <ActiveProjectProvider>
-                      <BottomSheetModalProvider>
-                        <PhotoPromiseProvider>
-                          <SecurityProvider>{children}</SecurityProvider>
-                        </PhotoPromiseProvider>
-                      </BottomSheetModalProvider>
-                    </ActiveProjectProvider>
-                  </ApiProvider>
-                </LocalDiscoveryProvider>
-              </ServerLoading>
-            </GPSModalContextProvider>
-          </TrackTimerContextProvider>
-        </GestureHandlerRootView>
+        <SafeAreaProvider>
```

Comment:
this fixes an issue where we were using `useSafeAreaInsets()` without a provider set up